asset/css/variables.less

```diff
@@ -48,7 +48,7 @@
 @state-down: @state-critical;
 @state-pending: #77aaff;
 @state-unknown: #aa44ff;
-@state-unreachable: @state-unknown;
+@state-unreachable: @gray-semilight;
```

Comment:
That's a variable from Icinga Web, use one that's defined here (preferred) or define a new one (but then don't forget the light mode variable)